recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - gcc  # [osx]
     - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
     - casacore 2.2.*
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
You will need to rerender again to update the build matrix. BTW, you can probably use:

`numpy 1.8.*` and `numpy >=1.8` for build and run respectively.